<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherIntroduce</class>
 <widget class="qMRMLWidget" name="TeacherIntroduce">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>226</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2002222</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="UnitTeachingIntro2" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>30</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>16777215</height>
         </size>
        </property>
        <widget class="QLabel" name="lblDes">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>0</y>
           <width>101</width>
           <height>26</height>
          </rect>
         </property>
         <property name="text">
          <string>教学讲师</string>
         </property>
        </widget>
        <widget class="QLabel" name="lblName">
         <property name="geometry">
          <rect>
           <x>28</x>
           <y>190</y>
           <width>100</width>
           <height>15</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="lblPic">
         <property name="geometry">
          <rect>
           <x>28</x>
           <y>46</y>
           <width>100</width>
           <height>140</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="lblIcon">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>26</width>
           <height>26</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>12</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>46</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>24</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>12</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="lblPatent">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnThesis">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>24</height>
               </size>
              </property>
              <property name="text">
               <string>查看论文</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnPatent">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>24</height>
               </size>
              </property>
              <property name="text">
               <string>查看专利</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblAdvantage">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
